src/js/components/SidebarFilter.js

```diff
@@ -136,12 +138,17 @@ class SidebarFilter extends mixin(QueryParamsMixin) {
   }
 
   getHealthCheckboxes() {
-    let {filterLabels, filterValues} = this.props;
+    let {filterLabels, filterType, filterValues} = this.props;
     let {selectedNodes} = this.state;
 
     return Object.keys(filterLabels)
       .filter(function (filterLabel) {
```

Comment:
Argh, yes, thx. :smile: 
I did unnecessary work.
I had this, but reverted it in the base branch, because of a comment.
https://github.com/dcos/dcos-ui/pull/762#discussion-diff-71377389
Funny.